astropy/wcs/src/wcslib_wrap.c

```diff
@@ -170,9 +301,7 @@ note_change(PyWcsprm* self) {
 }
 
 static void
-PyWcsprm_dealloc(
```

Comment:
Could you also undo these style-changes?

Reply:
I thought I did but maybe I missed some.